package.json

```diff
@@ -10,20 +10,23 @@
     "dist"
   ],
   "scripts": {
-    "build": "bili src/defu.ts",
+    "build": "siroc build",
     "lint": "eslint --ext .ts src",
     "prepublish": "yarn build",
     "release": "yarn test && yarn build && standard-version && git push --follow-tags && npm publish",
     "test": "yarn lint && yarn jest"
   },
+  "resolutions": {
+    "rollup-plugin-esbuild": "2.3.0"
```

Comment:
necessary to build on node 10 as after this point `rollup-plugin-esbuild` dropped node 10 support

Reply:
We can upgrade ci to use 12 or 14 :)